src/views/business/business-details.njk

```diff
@@ -145,18 +145,18 @@
             {% if hasVatNumber %}
               <ul class="govuk-summary-list__actions-list">
                 <li class="govuk-summary-list__actions-list-item">
-                  <a class="govuk-link" href="{{vatRemoveLink}}">
+                  <a class="govuk-link" href="/business-vat-registration-remove">
```

Comment:
These are now hard coded in the view but before they were dynamically passed from the presenter - should it stay being passed from the presenter?

Can see you have updated to use the new url in the presenter so feels like we should continue to pull the value from there?